cloud-init/docker-only-with-persistent-storage.txt

```diff
@@ -0,0 +1,79 @@
+#cloud-config
+
+coreos:
+  units:
+    - name: docker-tcp.socket
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=Docker Socket for the API
+
+        [Socket]
+        ListenStream=2375
+        BindIPv6Only=both
+        Service=docker.service
+
+        [Install]
+        WantedBy=sockets.target
+
+    - name: format-ephemeral.service
```

Comment:
i know ... fwiw i found that sub optimal as too wide reaching. i wanted the ability to take full advantage of the simplicity of PXE boot (as each time *everything* will work same way, no leftovers, etc) while having the ability to store persistently *just* what i wanted with *a la carte* granularity. (say ones want 1 vol for docker, another to  rkt, etc, to test version migrations/whatever). anyway will look onto extending current scheme to cover all angles. :-)